src/modules/cmdpal/Exts/Microsoft.CmdPal.Ext.Apps/NativeMethods.txt

```diff
@@ -0,0 +1,19 @@
+GetPhysicallyInstalledSystemMemory
```

Comment:
Probably good to note what we actually need here as a follow-up, not sure if CsWin32 just generates everything vs. optimizes what's actually referenced by the project.